bundle-workflow/src/test_workflow/test_result/test_suite_results.py

```diff
@@ -0,0 +1,26 @@
+from sortedcontainers import SortedDict  # type: ignore
+
+
+class TestSuiteResults(SortedDict):
+    def __init__(self):
+        super(TestSuiteResults, self).__init__()
+
+    def __append__(self, component, test_result_component):
+        self.__setitem__(component, test_result_component)
+
+    def append(self, component, test_result_component):
+        self.__append__(component, test_result_component)
+
+    def log(self):
+        for result in self.values():
+            result.log()
+
+    def status(self):
+        for result in self.values():
+            test_failed = result.failed
+            if test_failed:
+                return True
+        return False
```

Comment:
This method should also be called `failed`, not `status`, for the same reason as within each result. Status = true/false doesn't make a lot of sense.

I also believe this method can be written in 1 line: `return any(result.failed for result in self.values())`.